testing/src/cypress/e2e/02-institution/01-login-institution-page.cy.ts

```diff
@@ -11,25 +11,25 @@ const institutionHelperActions = new InstitutionHelperActions();
 const [URL, USERNAME, PASSWORD] =
   institutionHelperActions.getBaseUrlAndLoginCredentialsInstitution();
 
-describe("Login Page", () => {
+describe("[Institution Login] - Login Page", () => {
```

Comment:
Just asking for my understanding. Is there a specific reason of using Institution Login inside square brackets?

Reply:
That was just for readability sake and reporting purposes. But am removing as its already appended in the test results section